src/acl.c

```diff
@@ -1951,6 +2208,45 @@ void addACLLogEntry(client *c, int reason, int context, int argpos, sds username
  * ACL related commands
  * ==========================================================================*/
 
+void aclGetUserAddSelectorReply(client *c, aclSelector *s) {
```

Comment:
please add a top comment.

p.s. maybe a more appropriate name would be aclAddReplySelectorDescription?
the "Get" in the title made me think this method returns something to the caller.